AK/URLParser.cpp

```diff
@@ -1319,7 +1316,7 @@ URL URLParser::basic_parse(StringView raw_input, Optional<URL> const& base_url,
             url->m_scheme = "file";
 
             // 2. Set url’s host to the empty string.
-            url->m_host = "";
+            url->m_host = String();
```

Comment:
Nit, usually we prefer uniform initialization `String {}` (at least for classes and structs). This applies to all the `String()` occurrences in this commit.

Reply:
Ah, thanks, I'll fix that all up

Reply:
Done!